pom.xml

```diff
@@ -18,9 +18,42 @@
     <skipUnitTests>false</skipUnitTests>
   </properties>
 
+  <developers>
+    <developer>
+      <id>chingor</id>
+      <name>Jeff Ching</name>
+      <email>chingor@google.com</email>
+      <organization>Google</organization>
+      <roles>
+        <role>Developer</role>
+      </roles>
```

Comment:
Let's update this value. Maybe "Cloud Java team"?

Reply:
Would could try that, but I'm not sure that would be compliant with https://central.sonatype.org/publish/requirements/.

Reply:
I set you as the developer instead of Jeff.